testing/ceph/APKBUILD

```diff
@@ -0,0 +1,554 @@
+	mv "$pkgdir"$_bindir/cephfs-journal-tool "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/cephfs-table-tool "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/rados "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/rbd "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/rbd-replay "$subpkgdir"$_bindir/rbd-replay
+	mv "$pkgdir"$_bindir/rbd-replay-many "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/rbdmap "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/ceph-post-file "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/ceph-brag "$subpkgdir"$_bindir
+
+	mkdir -p "$subpkgdir"$_datadir/ceph
+	mv "$pkgdir"$_datadir/ceph/known_hosts_drop.ceph.com "$subpkgdir"$_datadir/ceph
+	mv "$pkgdir"$_datadir/ceph/id_rsa_drop.ceph.com "$subpkgdir"$_datadir/ceph
+	mv "$pkgdir"$_datadir/ceph/id_rsa_drop.ceph.com.pub "$subpkgdir"$_datadir/ceph
+
+	mkdir -p "$subpkgdir"$_sysconfdir/ceph
+	mv "$pkgdir"$_sysconfdir/ceph/rbdmap "$subpkgdir"$_sysconfdir/ceph
+
+	mkdir -p "$subpkgdir"$_python_sitelib
+	mv "$pkgdir"$_python_sitelib/ceph_argparse.py* "$subpkgdir"$_python_sitelib
+	mv "$pkgdir"$_python_sitelib/ceph_daemon.py* "$subpkgdir"$_python_sitelib
+
+	mkdir -p "$subpkgdir"$_udevrulesdir
+	mv "$pkgdir"$_udevrulesdir/50-rbd.rules "$subpkgdir"$_udevrulesdir
+
+	install -m 3770 -o ceph -g ceph -d "$subpkgdir"$_localstatedir/log/ceph
+	install -m 750 -o ceph -g ceph -d "$subpkgdir"$_localstatedir/lib/ceph
+}
+
+mds() {
+	pkgdesc="Metadata server daemon for the Ceph distributed file system."
+	depends="ceph-base"
+
+	mkdir -p "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/ceph-mds "$subpkgdir"$_bindir
+
+	install -m 750 -o ceph -g ceph -d "$subpkgdir"$_localstatedir/lib/ceph/mds
+}
+
+mon() {
+	pkgdesc="Cluster monitor daemon for the Ceph distributed file system."
+	depends="ceph-base"
+
+	mkdir -p "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/ceph-mon "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/ceph-rest-api "$subpkgdir"$_bindir
+
+	mkdir -p "$subpkgdir"$_python_sitelib
+	mv "$pkgdir"$_python_sitelib/ceph_rest_api.py* "$subpkgdir"$_python_sitelib
+
+	install -m 750 -o ceph -g ceph -d "$subpkgdir"$_localstatedir/lib/ceph/mon
+}
+
+ceph_fuse() {
+	pkgdesc="FUSE based client for Ceph distributed network file system."
+
+	mkdir -p "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/ceph-fuse "$subpkgdir"$_bindirn
+	mkdir -p "$subpkgdir"$_sbindir
+	mv "$pkgdir"$_sbindir/mount.fuse.ceph "$subpkgdir"$_sbindir
+}
+
+rbd_fuse() {
+	pkgdesc="FUSE based client to map Ceph rbd images to files."
+	depends="librados librbd"
+	mkdir -p "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/rbd-fuse "$subpkgdir"$_bindir
+}
+
+rbd_mirror() {
+	pkgdesc="Daemon for mirroring RBD images between Ceph clusters."
+	depends="ceph-common librados"
+	mkdir -p "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/rbd-mirror "$subpkgdir"$_bindir
+}
+
+rbd_nbd() {
+	pkgdesc="NBD based client to map Ceph rbd images to local device."
+	depends="librbd librados"
+	mkdir -p "$subpkgdir"$_bindir
+	mv "$pkgdir"$_bindir/rbd-nbd "$subpkgdir"$_bindir
+}
```

Comment:
I didn't look into your apkbuild too deeply, but you could try to limited the amount of repeating code by using a general function that accepts some options to split your packages.